06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -68,7 +67,7 @@ class InvertedIndex {
         const fileIndexed = this.getIndex(name);
         bookNames.push(fileIndexed.title);
         result.push(InvertedIndexUtility
-        .filterIndexed(query, fileIndexed.index));
+        .filterIndexed(searchTerms, fileIndexed.index));
       });
       searchResult = (fileNames.length !== 0) ?
```

Comment:
Please confirm this is the best way of using ternary operator. [See link.](https://github.com/airbnb/javascript#comparison--unneeded-ternary)